mezzanine/core/managers.py

```diff
@@ -175,8 +175,9 @@ def _clone(self, *args, **kwargs):
         """
         Ensure attributes are copied to subsequent queries.
         """
-        for attr in ("_search_terms", "_search_fields", "_search_ordered"):
-            kwargs[attr] = getattr(self, attr)
+        # i'm not quite sure what this is intended to do.
+        # for attr in ("_search_terms", "_search_fields", "_search_ordered"):
+            #kwargs[attr] = getattr(self, attr)
         return super(SearchableQuerySet, self)._clone(*args, **kwargs)
```

Comment:
I'd err on the side of this code actually doing something, let's not just blindly remove it.

Also please don't leave commented out code (mostly found with imports in this change).